tests/pytorch/r1.10/common.libsonnet

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import '../common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local volumes = import 'templates/volumes.libsonnet';
+
+local version = 'r1.10';
+{
+  PyTorchTest:: common.PyTorchTest {
+    frameworkPrefix: 'pt-%s' % version,
+    tpuSettings+: {
+      softwareVersion: 'pytorch-%s' % version,
+    },
+    imageTag: version,
+  },
+  PyTorchXlaDistPodTest:: common.PyTorchXlaDistPodTest {
+    frameworkPrefix: 'pt-%s' % version,
+    tpuSettings+: {
+      softwareVersion: 'pytorch-%s' % version,
+    },
+    imageTag: version,
+  },
+  PyTorchGkePodTest:: common.PyTorchGkePodTest {
+    frameworkPrefix: 'pt-%s' % version,
+    tpuSettings+: {
+      softwareVersion: 'pytorch-%s' % version,
+    },
+    imageTag: version,
+  },
+  Functional:: mixins.Functional {
+    schedule: '30 5 * * *',
+    tpuSettings+: {
+      preemptible: false,
+    },
+  },
+  Convergence:: mixins.Convergence {
+    // Run 3 times/week.
+    schedule: '0 7 * * 1,3,5',
+  },
+  datasetsVolume: volumes.PersistentVolumeSpec {
+    name: 'pytorch-datasets-claim',
+    mountPath: '/datasets',
+  },
+  tpu_vm_1_10_install: |||
+    sudo bash /var/scripts/docker-login.sh
+    sudo docker rm libtpu || true
+    sudo docker create --name libtpu gcr.io/cloud-tpu-v2-images/libtpu:pytorch-1.9 "/bin/bash"
```

Comment:
this is on purposely kept at 1.9, we'll update to 1.10 once tf build is successful.
besides newest libtpu is installed by pip, will remove these couple of lines that deals with libtpu.so file
please let me know if it should still be otherwise, thank you!